autorest/codegen/models/operation.py

```diff
@@ -251,7 +246,7 @@ def imports(self, code_model, async_mode):
             file_import.add_import("uuid", ImportType.STDLIB)
 
         # Deprecation
-        if True:  # Replace with "the YAML contains deprecated:true"
+        if True:  # Replace with "the YAML contains deprecated:true"  # pylint: disable=using-constant-test
```

Comment:
If think if you add a FIXME at the beginning of the comment, it disables the pylint error, and VSCode can list FIXME automatically. I would suggest that.

Reply:
Adding a comment with FIXME isn't getting rid of the pylint error, so I'll add a pylint disable, but also add FIXME to the comment for VSCode